src/main/java/core/basesyntax/strategy/handler/IncrementHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.OperationHandler;
+
+public abstract class IncrementHandler implements OperationHandler {
+    @Override
+    public void apply(Fruit fruit, int quantity) {
+        fruit = Storage.fruits.getOrDefault(fruit.getName(), new Fruit(fruit.getName()));
+        fruit.addQuantity(quantity);
+        Storage.fruits.put(fruit.getName(), fruit);
+    }
+}
```

Comment:
According to the checklist, we should follow the SOLID principles. The IncrementHandler is marked as abstract, but it does not contain any abstract methods and is fully implemented. This could potentially violate the Open/Closed principle if we need different behaviors for incrementing. Consider whether this class should be abstract or if it should be a concrete class with the potential to be extended or modified through other means (like composition).